### PR TITLE
FLUID-6288: use Docker containers pinned to Node 8

### PR DESCRIPTION
**Pull request by waharnum**
As discussed in-channel and in-person, this uses Docker containers pinned to Node Version 8 to avoid having to upgrade to the DocPad 6.8 line that we've been fighting with off and on for several months due to plugin compatibility issues and other problems.

### CODE REVIEW

**Comment by waharnum** [comment]
@simonbates, for your review when possible (you'll need Docker locally, but I believe you have this installed).

File: README.md [hunk 1]
```diff
@@ -1,10 +1,48 @@
...
 You can [browse the Infusion Documentation as static user friendly pages](http://docs.fluidproject.org/infusion/), or if you prefer, you can [browse the Infusion Documentation directly on GitHub](src/documents).
 
-## Build with DocPad
+## Docker Instructions
+
+Docker provides us with an environment to run Node 8 in that is separated from the host, in order to work on the site. Since we deploy the static product of the DocPad build process, this is OK for now.
+
+### Docker Dev Container (for local development purposes)
+
+Builds ands run site in DocPad, with a bind mount between local `src` and `src in container`, allowing for local development.
```
Comment (simonbates) [inline README.md]: Minor typo: "Builds ands run site..." should be "Builds and runs site...".

File: Dockerfile-dev [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+# Switch to regular user
+USER node
+
+# Install npm dependencies
+COPY --chown=node package.json /app/package.json
+WORKDIR /app
+RUN npm install
+
+# Build and test website
```
Comment (simonbates) [inline Dockerfile-dev]: I think this comment is not correct

**Comment by waharnum** [comment]
As discussed, I'm going to open a new JIRA and PR for explaining how to run Docpad 6.79.4 using Node 10 - the most bare-bones solution to the present issue.

**Comment by jobara** [comment]
We're in the process of migrating from DocPad to 11ty. I'm closing this issue as it is related to a specific problem we are encountering with DocPad.

**Pull request closed without merging**